modules/html/src/example.json

```diff
@@ -1,12 +1,5 @@
 {
-    "name": "__bold",
-    "arguments": {},
-    "children": [
-        {
-            "name": "__text",
-            "data": "Hello, world",
-            "arguments": {},
-            "inline": true
-        }
-    ]
-}
\ No newline at end of file
+    "name": "__error",
```

Comment:
We might want to revert this change (or maybe just remove the `example.json` file)

Reply:
Yes, I will revert it